proxy/envoy/config_test.go

```diff
@@ -73,26 +74,52 @@ func TestRoutesByPath(t *testing.T) {
 }
 
 const (
-	envoyData   = "testdata/envoy.json"
-	envoyGolden = "testdata/envoy.json.golden"
+	envoyData          = "testdata/envoy.json"
+	//envoyGolden        = "testdata/envoy.json.golden"
+	envoyTimeoutGolden = "testdata/envoy-timeout.json.golden"
+	routeRule          = "testdata/route-rule.json.golden"
 )
 
 func TestMockConfigGenerate(t *testing.T) {
 	ds := mock.Discovery
 	r := mock.MakeRegistry()
+
+	rTemp, err := ioutil.ReadFile(routeRule)
```

Comment:
This is a good idea - was going to do that once your PR is in. I'd suggest adding another golden config with all rules applied. That way it's easier to see problems with core TCP/HTTP routing vs policies vs filters.